check_borg

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash
+#!/bin/sh
+set -o errexit
+set -o nounset
 
 PROGNAME=$(basename "$0")
-PROGPATH=$(echo "$0" | sed -e 's,[\\/][^\\/][^\\/]*$,,')
+PROGPATH=$(dirname $(readlink -f $0))
```

Comment:
The previous `PROGPATH` instruction had been copied from nagios/monitoring plugins.

There is a fundamental difference to yours.

If you link the script into `/usr/lib/nagios/plugins` (or the respective libexec folder). `PROGPATH` is different. And in this case, I doubt it's right to reference the actual folder.

---

But anyway. Why is there `PROGPATH`? We don't need it! Please remove it. So nobody will think about fixing/using this in future again. I guess we don't need any static extra files to read before checking borgbackup.

Reply:
Okay, I didn't see that use-case (linking to nagios-plugins).

PROPATH is being used a few lines later. I'll just remove that commit.